client/src/infographic/Infographic.js

```diff
@@ -422,8 +422,32 @@ const Infographic = ({
       <Redirect to={infograph_href_template(subject_parent, bubble_id, "/")} />
     );
   }
-
   const title = text_maker("infographic_for", { subject });
+  const breadcrumbs = (() => {
+    const get_breadcrumbs = (parent_subjects) =>
+      _.chain(parent_subjects)
+        .map((parent_subj) => (
+          <a
+            href={infograph_href_template(parent_subj, active_bubble_id)}
```

Comment:
I was gonna make this active bubble id smart (services exist for program but not for crso, etc) but figured I'd leave it as is since typeahead behaves the same way and couldn't think of a fully automated way